CredentialCardList.vue

```diff
@@ -106,6 +104,7 @@ $breakpoint-xs: 320px;
   border-radius: 6px;
   min-height: 83px;
   width: 100%;
+  font-size: 1rem;
```

Comment:
Dunno about using rems here -- but going to ignore this for now.

Reply:
Normally I would not but this was previously the default for Quasar so I set this to match what it was before. We can use pixels in you prefer.

Reply:
We can leave it for now. We'll find out later if it's a problem and can adjust.